From 6bbf3a56fb4cdae27763d9a6a40d490173f028b1 Mon Sep 17 00:00:00 2001
From: Oleksandr Tyshchenko <oleksandr_tyshchenko@epam.com>
Date: Tue, 5 Nov 2019 15:09:12 +0200
Subject: [PATCH] Add vspfilter config for salvator-x-4x2g machine

It should be noted that we use other /dev/videoX nodes in comparison
with DomD. The reason is that corresponding VSP devices enumerated
differently.

cat /sys/class/video4linux/video14/name
   fe9b0000.vsp rpf.0 input
cat /sys/class/video4linux/video15/name
   fe9b0000.vsp wpf.0 output

Signed-off-by: Oleksandr Tyshchenko <oleksandr_tyshchenko@epam.com>
---
 .../gstvspfilter-salvator-x-4x2g_r8a7795.conf                           | 2 ++
 1 file changed, 2 insertions(+)
 create mode 100644 meta-rcar-gen3/recipes-multimedia/gstreamer/gstreamer1.0-plugin-vspfilter/gstvspfilter-salvator-x-4x2g_r8a7795.conf

diff --git a/meta-rcar-gen3/recipes-multimedia/gstreamer/gstreamer1.0-plugin-vspfilter/gstvspfilter-salvator-x-4x2g_r8a7795.conf b/meta-rcar-gen3/recipes-multimedia/gstreamer/gstreamer1.0-plugin-vspfilter/gstvspfilter-salvator-x-4x2g_r8a7795.conf
new file mode 100644
index 0000000..74a8ab9
--- /dev/null
+++ b/meta-rcar-gen3/recipes-multimedia/gstreamer/gstreamer1.0-plugin-vspfilter/gstvspfilter-salvator-x-4x2g_r8a7795.conf
@@ -0,0 +1,2 @@
+input-device-name=/dev/video14
+output-device-name=/dev/video15
-- 
2.7.4

